packages/truffle-box/lib/config.ts

```diff
@@ -1,6 +1,6 @@
-const fse = require("fs-extra");
+import fse from "fs-extra";
 
```

Comment:
Probably worth adding an `export interface BoxConfig { }` and making `read` return that, so people can see what they should expect.